conf/machine/ov-cubieboard2.inc

```diff
@@ -0,0 +1,19 @@
+# Common machine definitions for all Cubieboard2-based OpenVarios
+
+PREFERRED_VERSION_linux-mainline ?= "5.10.%"
+
+require conf/machine/include/sunxi.inc
+#require conf/machine/include/sunxi-mali.inc
+require conf/machine/include/tune-cortexa7.inc
+
+IMAGE_CLASSES += "sdcard_image-openvario"
+IMAGE_FSTYPES = "openvario-sdimg"
+
+SOC_FAMILY = "sun7i"
+
+KERNEL_DEVICETREE = "openvario.dtb"
+#KERNEL_DEVICETREE = "sun7i-a20-cubieboard2.dtb"
+UBOOT_MACHINE = "openvario_defconfig"
+SUNXI_FEX_FILE = "sys_config/a20/cubieboard2.fex"
```

Comment:
This line was copied from meta-sunxi/cubieboard2.conf, and my PR *removes* it; instead of copying stuff from other repositories, meta-openvario should inherit (and extend) those files. Copying stuff is rarely ever a good idea; it causes big problems when updating to new Yocto versions.